server/service.go

```diff
@@ -319,6 +319,17 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 		return
 	}
 
+	// If genesisTime and slotTimeSec are set, log how late into the slot the request started
+	if config.GenesisTime > 0 && config.SlotTimeSec > 0 {
```

Comment:
cant we log without this check ? if the default is them being set to the constants, then it feels like we should always log.

Reply:
sure. this was intended as a guard against user overwriting this with weird times, but that seems overly cautious and not necessary

Reply:
that or we make `GetEnvUint` utils, and only allow unsigned ints

Reply:
it already is overwritable through GetEnvUint